.github/workflows/ion-java-performance-regression-detector.yml

```diff
@@ -0,0 +1,127 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Ion Java performance regression detector
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
> `push: branches: [master]` seems redundant here since `amzn/master` always require a pull request before pushing changes into it (by its branch protection rule).

Good catch, push: branches: [master] will be deleted in the next commit.